### PR TITLE
BUGFIX: SD-826 SD-862 fixed layout tooltip

### PR DESCRIPTION
**Pull request by dlcd2203**
---

[Check PR owner responsibilities](https://confluence.intgdc.com/display/Development/Code-reviews#Code-reviews-Ownerresponsibilities)

Supported PR commands:

Command | Description
--- | ---
`extended test - examples` | Live examples tests
`extended test - storybook` | Storybook screenshot tests

See more [options](https://confluence.intgdc.com/display/kbhr/How+to+work+with+git+and+Github#HowtoworkwithgitandGithub-Extendedchecks).

# PR Checklist

- [x] Verify code changes ([Checklist](https://confluence.intgdc.com/display/Development/Code-reviews+checklist), [Best practices](https://confluence.intgdc.com/display/Development/Code-reviews+best+practices))
- [x] [Verify pull-request formalities](https://confluence.intgdc.com/display/Development/Code-reviews)
- [x] Change was tested by using [gdc-dev-release](https://confluence.intgdc.com/display/~tomas.vojtasek/Private+NPM) in [gdc-analytical-designer](https://github.com/gooddata/gdc-analytical-designer) and [gdc-dashboards](https://github.com/gooddata/gdc-dashboards) (if applicable)
- [x] Migration guide (for major changes) is mentioned in [CHANGELOG.md](../blob/master/CHANGELOG.md).
- [x] Successful `extended test - examples`
- [x] Successful `extended test - storybook`
- [x] Checked yarn.lock consistency (no dep. duplicities especially Goodstrap)

# Related PRs

- gdc-analytical-designer: https://github.com/gooddata/gdc-analytical-designer/pull/3122
- gdc-dashboards: https://github.com/gooddata/gdc-dashboards/pull/2838

# Related Jira tasks

SD-826: https://jira.intgdc.com/browse/SD-826
SD-862: https://jira.intgdc.com/browse/SD-862

### CODE REVIEW

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/7473/ : SUCCESS in 6m 11s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/3516/ : SUCCESS in 3s

**Comment by dlcd2203** [comment]
extended test - storybook

**Comment by yenkins** [comment]
Link to Storybook test result:

https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/3915/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/3544/ : SUCCESS in 3s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/3272/ : SUCCESS in 9m 56s

**Comment by dlcd2203** [comment]
extended test - examples

**Comment by yenkins** [comment]
:arrow_right: Started Live Examples deploy to:
https://gdc-live-examples-1619.herokuapp.com

It will be destroyed on merge/close. See [ci-infra](https://github.com/gooddata/ci-infra/blob/master/jenkins/jobs/templates/orchestrators/client-pr-dispatcher.groovy) and [gdc-ci.yaml](https://github.com/gooddata/gooddata-react-components/blob/master/gdc-ci.yaml).

**Comment by ptquang86** [comment]
LGTM

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/3545/ : SUCCESS in 3s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/3273/ : SUCCESS in 17m 24s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/7487/ : SUCCESS in 5m 08s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/3548/ : SUCCESS in 3s

**Comment by naibinh** [comment]
LGTM

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/7506/ : SUCCESS in 2m 49s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/3577/ : SUCCESS in 3s

File: src/components/visualizations/utils/layout.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
+// (C) 2020 GoodData Corporation
```
Comment (lanhq) [inline src/components/visualizations/utils/layout.ts:1]: i would move this change to `src/components/visualizations/chart/tooltip.ts`

Comment (dlcd2203) [reply src/components/visualizations/utils/layout.ts]: fixed

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/7529/ : SUCCESS in 2m 55s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/3614/ : SUCCESS in 3s

**Comment by dlcd2203** [comment]
extended test - storybook

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/3942/ : ABORTED in 51s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-fossa-scanning-zuul-el7/41/ : SUCCESS in 20s (non-voting)
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/7530/ : ABORTED in 1m 59s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/1816/ : SUCCESS in 13s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-examples-zuul-docker/2679/ : ABORTED in 2m 15s

**Comment by gdgate** [comment]
Build in pipeline check-extended aborted.

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/7531/ : SUCCESS in 2m 46s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/3616/ : SUCCESS in 3s

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/3944/ : SUCCESS in 10m 10s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-fossa-scanning-zuul-el7/42/ : SUCCESS in 16s (non-voting)
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/7532/ : SUCCESS in 2m 34s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/1817/ : SUCCESS in 11s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-examples-zuul-docker/2680/ : SUCCESS in 15m 48s

**Pull request merged**